.github/workflows/avm.res.event-grid.domain.yml

```diff
@@ -0,0 +1,81 @@
+name: "avm.res.event-grid.domain"
```

Comment:
Hey @fabmas, please note that because this workflow does not exist in main, it won't show. My recommendation (though not perfect) for a general flow is:
1. Create the pipeline in the `main` branch
2. Then create your development branch based on main in which you develop the module 
3. Open your PR from this branch

The only but important caveat of this approach is that if you develop multiple modules at the same time, you have to remove those workflows that don't exist in upstream yet (i.e., the workflows of your other modules you work on) from this dev branch (but not `main`) after creating it.
If you do that, you'll have
- a `main` branch with all workflows
- a `module1` branch with only the `module1` workflow
- a `module2` branch with only the `module2` workflow
- ...

I hope that helps and made some sense. Ain't easy to explain without drawing a picture :D